packit_service/mapping.py

```diff
@@ -0,0 +1,22 @@
+from typing import Dict, Optional
```

Comment:
this file is really meta and complicated, but I'm fine with it if you are fine

Reply:
I'll add some comments and the licence header.

Reply:
I'll add some comments and the licence header.